src/content/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more.mdx

```diff
@@ -189,7 +189,13 @@ Our default functionality is to manage user type in New Relic. This is the optio
     title="Manage user type with SCIM"
   >
 
-If you've [enabled SCIM](#source-users) in the authentication domain UI, you can use our [SCIM API](/docs/accounts/accounts/automated-user-management/tutorial-manage-users-groups-scim#manage-user-type) to manage user type. (This is not yet available for the [Azure AD](/docs/accounts/accounts/automated-user-management/azure-ad-scimsso-application-configuration), [Okta](/docs/accounts/accounts/automated-user-management/okta-scimsso-application-configuration), or [OneLogin](/docs/accounts/accounts/automated-user-management/onelogin-scimsso-application-configuration) New Relic apps.) The upgrade request UI allows you to create a custom URL and message that basic users will see when they go to New Relic features that are off limits to them, and that message will help them understand your specific process for upgrading. 
+Our [SCIM API](/docs/accounts/accounts/automated-user-management/tutorial-manage-users-groups-scim#manage-user-type) lets you manage user type (basic user versus full user) from your identity provider instead of in the New Relic user management UI. When you enable this, you can't manage user type from the New Relic side.
+
+Currently the only identity provider we have instructions for is [Okta](/docs/accounts/accounts/automated-user-management/okta-scimsso-application-configuration). For other identity provider services, you can use our [SCIM API instructions for sending `nrUserType`](/docs/accounts/accounts/automated-user-management/tutorial-manage-users-groups-scim/#manage-user-type).
+
+Notes on using this: 
+* To enable this setting, you must first [enable SCIM](#source-users) under the **Source of users**. 
+* When you enable this setting, you're prompted to create a custom URL and message that basic users see when they go to New Relic features that are off limits to them. That message will help them understand your specific process for upgrading. 
```

Comment:
"features that are off limits to them." seems like it should be a separate sentence. "These features are off limits to them."